bundle-hack/update_bundle.sh

```diff
@@ -1,5 +1,119 @@
 #!/usr/bin/env bash
 
-echo "Bundle updated!"
+# This update_bundle script was inspired from the external-dns-operator render templates on CPaaS with the intention of minimal changes during Konflux migration
+# https://gitlab.cee.redhat.com/cpaas-midstream/external-dns-operator/-/blob/ext-dns-optr-1.3-rhel-9/distgit/containers/external-dns-operator-bundle/render_templates?ref_type=heads
+set -x
+set -e
+
+source ./container_digest.sh
+source ./bundle_vars.sh
+
+export VERSION=$(cat VERSION)
```

Comment:
`bundle_vars.sh` script needs `VERSION` variable, so it has to be exported before.